ui/desktop/electron-app/src/index.js

```diff
@@ -152,28 +82,22 @@ app.on('ready', async () => {
       'icon.png'
     );
 
-  mainWindow = new BrowserWindow(browserWindowOptions);
+  const win = new BrowserWindow(browserWindowOptions);
```

Comment:
```suggestion
  const browserWindow = new BrowserWindow(browserWindowOptions);
```

Reply:
I did not want it to call it `browserWindow` to avoid confussion with Electron API `BrowserWindow`, but since that was just a personal thought, performing this change.